source/math/Quaternion.ooc

```diff
@@ -78,7 +79,10 @@ Quaternion: cover {
 		z := a1 * d2 + b1 * c2 - c1 * b2 + d1 * a2;
 		return This new(w, x, y, z);
 	}
-	fromRotationMatrix: static func (matrix: FloatTransform2D) -> This {
+	// TODO: This cannot be used with FloatTransform2D, only with a 3D rotation matrix
+	// for which we don't have a data type. We could do this for FloatTransform3D but that would
+	// make the calculations more complex than we have here.
```

Comment:
So... a Quaternion can only represent rotations, not translations. So, should it be possible to verify that the incoming matrix is a rotation matrix first and then perform the conversion...?